test/fixtures/minify-single-line-comments-with-interpolations/output.js

```diff
@@ -1,29 +1,29 @@
 import styled from 'styled-components';
 const Test1 = styled.div.withConfig({
-  displayName: "code__Test1",
-  componentId: "kc0mjf-0"
+  displayName: "sc-code__Test1",
```

Comment:
The `displayName` shouldn't be getting prefixed

Reply:
Right, I see. That may be why the bug was first introduced. By only prefixing digit, the goal was to skip prefixing the displayName, but it also skipped prefixing some of the hash. I'll look into it.

Reply:
@probablyup I've updated to prevent the prefixing of the displayName.